Sensus.Shared.iOS/Callbacks/UILocalNotifications/UILocalNotificationNotifier.cs

```diff
@@ -37,22 +39,6 @@ public override void IssueNotificationAsync(string message, string id, bool play
         {
             SensusContext.Current.MainThreadSynchronizer.ExecuteThreadSafe(() =>
             {
-                // cancel any existing notifications with the given id
-                lock (_notifications)
-                {
-                    foreach (UILocalNotification notification in _notifications.ToList())
-                    {
-                        string notificationId = notification.UserInfo.ValueForKey(new NSString(NOTIFICATION_ID_KEY)).ToString();
-                        if (notificationId == id)
-                        {
-                            CancelNotification(notification, NOTIFICATION_ID_KEY);
-
-                            // TODO:  How do we prevent this collection from growing without bound?
-                            _notifications.Remove(notification);
-                        }
-                    }
-                }
-
```

Comment:
Yeah I think in this case I definitely introduced a bug. I think we should simply put in CallbackId and go with that. Thoughts?